core/net/rpl/rpl-icmp6.c

```diff
@@ -86,6 +90,7 @@ void RPL_DEBUG_DAO_OUTPUT(rpl_parent_t *);
 #endif
 
 static uint8_t dao_sequence = RPL_LOLLIPOP_INIT;
+static uint8_t downward = 0;
```

Comment:
Yes, and I will move it to the instance also as it in theory could be multiple active instances with only one of them having a downward route.